pkgs/top-level/all-packages.nix

```diff
@@ -697,6 +697,8 @@ in
     optgamsFile = config.gams.optgamsFile or null;
   };
 
+  gita = callPackage ../applications/version-management/gita {};
```

Comment:
I think this should be moved to `gitAndTools`, i.e. `pkgs/applications/version-management/git-and-tools/default.nix`, and the expression should be moved to `pkgs/applications/version-management/git-and-tools/gita/default.nix`.